src/providers/codeAction/swapListItemCodeAction.ts

```diff
@@ -0,0 +1,161 @@
+import {
+  CodeAction,
+  CodeActionKind,
+  Range,
+  TextEdit,
+} from "vscode-languageserver";
+import { SyntaxNode } from "web-tree-sitter";
+import { PositionUtil } from "../../positionUtil";
+import { TreeUtils } from "../../util/treeUtils";
+import {
+  CodeActionProvider,
+  IRefactorCodeAction,
+  IRefactorEdit,
+} from "../codeActionProvider";
+import { ICodeActionParams } from "../paramsExtensions";
+
+// Handle adding annotation to let expr declaration
+const refactorName = "swap_listitem";
+const moveListItemUpActionName = "swap_listitem_up";
+const moveListItemDownActionName = "swap_listitem_down";
+CodeActionProvider.registerRefactorAction(refactorName, {
+  getAvailableActions: (params: ICodeActionParams): IRefactorCodeAction[] => {
+    const nodeAtPosition = TreeUtils.getNamedDescendantForPosition(
+      params.sourceFile.tree.rootNode,
+      params.range.start,
+    );
+
+    const canMoveNext = getTargetNodes(nodeAtPosition, "next");
+    const canMovePrev = getTargetNodes(nodeAtPosition, "previous");
+
+    const codeActions: IRefactorCodeAction[] = [];
+
+    if (canMovePrev) {
+      codeActions.push({
+        title: "Move List item Up",
+        kind: CodeActionKind.RefactorRewrite + ".movelistitem.up",
+        data: {
+          actionName: moveListItemUpActionName,
+          refactorName,
+          uri: params.sourceFile.uri,
+          range: params.range,
+        },
+      });
+    }
+
+    if (canMoveNext) {
+      codeActions.push({
+        title: "Move List item Down",
```

Comment:
```suggestion
        title: "Move list item down",
```
I would expect these to be cased like this